man/tmpfiles.d.xml

```diff
@@ -243,15 +243,15 @@ L     /tmp/foobar -    -    -     -   /dev/null</programlisting>
         <varlistentry>
           <term><varname>L</varname></term>
           <term><varname>L+</varname></term>
-          <listitem><para>Create a symlink if it does not exist
-          yet. If suffixed with <varname>+</varname> and a file or
-          directory already exists where the symlink is to be created,
-          it will be removed and be replaced by the symlink. If the
-          argument is omitted, symlinks to files with the same name
-          residing in the directory
-          <filename>/usr/share/factory/</filename> are created. Note
-          that permissions and ownership on symlinks are ignored.
-          </para></listitem>
+          <listitem><para>Create a symbolic link (symlink) if it does not exist yet. If the command character
+          is suffixed with <varname>+</varname> and a file or directory already exists where the symlink is
+          to be created, it will be removed and be replaced by the symlink. The symlink target is defined in
+          the argument column. If the argument is omitted, symlinks to files with the same name residing in
+          the directory <filename>/usr/share/factory/</filename> are created. Note that permissions and
+          ownership on symlinks are ignored. If specified the symlink target is used literally and is
+          unaffected by the <option>--root=</option> command line option of
+          <command>systemd-tmpfiles</command>. In order to create symlinks that take <option>--root=</option>
+          into account, consider generating relative rather than absolute symlinks.</para></listitem>
```

Comment:
This description is wrong / misleading. For `L` links, we create a symlink with exactly the contents specified in the argument, both for arguments that are relative and absolute. So this last sentence with "consider generating relative rather than absolute symlinks" makes no sense.

I think current behaviour is correct: there are two modes of operation for tmpfiles: one is when configuration is passed explicitly on the command line (as file names or as inline arguments), and one where the configuration files are read from the standard set of directories. In this second mode, `--root` affects where the files are sourced from. But if those files contains `L` lines, those lines only make sense relative to `--root`. If we would make them relative to caller's root, suddenly we loose the property that operation with `--root` is the same as doing `chroot` and then calling `systemd-tmpfiles` without the `--root` argument.

For example, if I have
```
# /etc/tmpfiles.d/my.conf
L /etc/os-release - - - - /usr/lib/os-release
L /etc/os-release2 - - - - ../usr/lib/os-release
```
in and I call systemd-tmpfiles with `--root`, I'd expect to get two symlinks that make sense in the container, and they should both point to the same file in the container's /usr/lib. This is what currently happens.